src/number_theory/padics/padic_norm.lean

```diff
@@ -153,13 +153,13 @@ else if hr : r = 0 then
   by simp [hr]
 else
   have q*r ≠ 0, from mul_ne_zero hq hr,
```

Comment:
```suggestion
  have q * r ≠ 0, from mul_ne_zero hq hr,
```
Why not, even if you didn't modify this line.

Reply:
Same here